src/photos.cs

```diff
@@ -1175,6 +1190,15 @@ interface PHPhotoLibrary {
 		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
 		[Field ("PHLocalIdentifiersErrorKey")]
 		NSString LocalIdentifiersErrorKey { get; }
+
+		[TV (16,0), Mac (13,0), iOS (16,0)]
+		[Export ("fetchPersistentChangesSinceToken:error:")]
+		[return: NullAllowed]
+		PHPersistentChangeFetchResult FetchPersistentChanges (PHPersistentChangeToken token, [NullAllowed] out NSError error);
```

Comment:
I think this captures the selector name better:

```suggestion
		PHPersistentChangeFetchResult FetchPersistentChanges (PHPersistentChangeToken since, [NullAllowed] out NSError error);
```

Reply:
Fixed.